service/controller/resource/heartbeatrouting/receiver/receiver_test.go

```diff
@@ -179,6 +146,60 @@ func TestEnsureReceiver(t *testing.T) {
 	}
 }
 
+func TestEnsureReceiverWithProxy(t *testing.T) {
+	// The HTTPConfig from global config should be merged when creating
+	// HTTPConfig for the receiver
```

Comment:
Suggestion for a better comment, making it a valid go comment is my main point here.
```suggestion
// TestEnsureReceiverWithProxy ensures the global HTTPConfig is merged into receiver HTTPConfig.
func TestEnsureReceiverWithProxy(t *testing.T) {
```

Reply:
Lol, I meant to do that and it slipped my mind, thanks!